roles/project/tasks/main.yml

```
@@ -0,0 +1,42 @@
+---
+
+# Create EDA Controller Project
+- name: Add EDA Controller project
+  eda_project:
```

```suggestion
  project:
```

This should be project. The module is called that